kytos/core/kytosd.py

```diff
@@ -102,11 +102,15 @@ def main():
 
     config = KytosConfig().options['daemon']
 
-    if config.foreground:
-        async_main(config)
-    else:
-        with daemon.DaemonContext():
+    if config.daemon:
+        if config.foreground:
             async_main(config)
+        else:
+            with daemon.DaemonContext():
+                async_main(config)
+    else:
+        config.foreground = True
+        async_main(config)
```

Comment:
I think it can be simplified into this:
```suggestion
    if config.foreground or not config.daemon:
        async_main(config)
    else:
        with daemon.DaemonContext():
            async_main(config)
```